src/main/java/org/opentripplanner/routing/edgetype/StreetEdgeTraversal.java

```diff
@@ -243,14 +274,22 @@ public StateEditor doTraverse(RoutingRequest options, TraverseMode traverseMode)
                 // TODO: how to weight off of these values?
             }
 
+            if (options.wheelchairAccessible && curbRamp == CurbRamp.NO) {
```

Comment:
Should this be just `if (options.wheelchairAccessible)`? I'm not quite clear what the purpose of the check on `curbRamp` is.